### PR TITLE
More mock bids, half of which are submitted

### PR DESCRIPTION
**Pull request by mjoyce91**

### CODE REVIEW

File: seeds/016_bids_seed.js [hunk 1]
```diff
@@ -15,15 +15,27 @@ exports.seed = function(knex) {
...
+                        cdo_bid: randCdoBid === 1 ? 'Y' : 'N',
+                      }
+                    }
+                    bids.push(bid)
                   }
                 })
               })
-              return knex('bids').insert(bids);
+              // run in batches of 1000 to prevent 08P01 error
+              return knex.batchInsert('bids', bids, 1000)
```
Comment (elizabeth-jimenez) [inline seeds/016_bids_seed.js:38]: 😎

**Review by elizabeth-jimenez — APPROVED** [review]
🍬 👍

File: seeds/016_bids_seed.js [hunk 2]
```diff
@@ -15,15 +15,27 @@ exports.seed = function(knex) {
...
+                  const randSubmit = Math.floor(Math.random() * 2) // 50% chance to submit the bid
+                  const randCdoBid = Math.floor(Math.random() * 4) // 25% chance it was a CDO bid
                   if (rand === 1) {
-                    bids.push({
+                    let bid = {
                       perdet_seq_num: emp.perdet_seq_num,
                       cp_id: cpid['cp_id'],
-                    })
+                    }
+                    if (randSubmit === 1) {
```
Comment (elizabeth-jimenez) [inline seeds/016_bids_seed.js:25]: could change to `if (randSubmit) {` if ya want to

Comment (mjoyce91) [reply seeds/016_bids_seed.js:25]: Keeping it 1 means we could change the percentage chance above and not need to change this back to 1

Comment (elizabeth-jimenez) [reply seeds/016_bids_seed.js:25]: ohh good point!

**Pull request merged**